firefox-ios/firefox-ios-tests/Tests/ClientTests/LiteLLMClientTests.swift

```diff
@@ -0,0 +1,148 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+@testable import Client
+import XCTest
+import Common
+import Shared
+
+class LiteLLMClientTests: XCTestCase {
```

Comment:
```suggestion
final class LiteLLMClientTests: XCTestCase {
```

thanks for writing tests!! It also makes it easier to see what the payloads / responses will look like. Would be nice to test all the other `public` methods as well if we can.